ParkBlazerApp/src/components/map.tsx

```diff
@@ -142,6 +150,21 @@ export default function Map({ onUpdateList, onLocationMarkerUpdate }: any) {
               <br></br>
               <IonText><strong>Lastwagen:</strong> <IonIcon icon={selectedSpot.type_truck ? checkmark : close} /></IonText>
             </IonCard>
+            {isPrivateModal && (
+              <>
+                <IonCard>
+                  <IonText><strong>Preis:</strong> {selectedSpot.price_per_hour} €</IonText>
+                </IonCard>
+                {selectedSpot.availability_start_date && selectedSpot.availability_end_date && (
+                  <IonCard>
+                  <IonText><strong>Verfügbarkeit:</strong></IonText>
+                  <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center', padding: '20px' }}>
```

Comment:
Wenn man so viel stylen muss, sollte man das nicht inline machen. Bitte eine id geben und in die css Datei verschieben